homeassistant/components/aqara/.gitignore

```diff
@@ -0,0 +1,140 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This file seems like it's left over from this being a stand-alone custom component?
I think it should be removed.

Reply:
Yup, this file needs to be removed.

Reply:
removed